Makefile

```diff
@@ -314,4 +314,4 @@ openshiftci-presubmit-unittests:
 # Run OperatorHub tests
 .PHONY: test-operator-hub
 test-operator-hub:
-	ginkgo $(GINKGO_FLAGS) -focus="odo service command tests" tests/integration/operatorhub/
\ No newline at end of file
+	ginkgo $(GINKGO_FLAGS_SERIAL) -focus="odo service command tests" tests/integration/operatorhub/
```

Comment:
why are we running this test in serial?

Reply:
Because running them in parallel leads to https://github.com/openshift/odo/pull/2901#issuecomment-616960021. There are two specs that spin up an operator backed service using the same yaml and hence have same value for `metadata.name`.

Reply:
Can't we use different values for those tests?

Reply:
I could do some string manipulation and change it to `example2`. But only if that would be OK. There are longer ways to do it but it'd be over-engineering because I'm also doing regex match based on it.

Reply:
Let's try the string manipulation

Reply:
I tried `strings.Replace` but that's messing up the yaml formatting. So the next option could be to do yaml unmarshalling. It's a pain, tbh.

Reply:
This works. Would it be OK? @kadel @amitkrout @girishramnani 
```diff
$ git diff
diff --git a/tests/integration/operatorhub/cmd_service_test.go b/tests/integration/operatorhub/cmd_service_test.go
index afe598791..9797219a2 100644
--- a/tests/integration/operatorhub/cmd_service_test.go
+++ b/tests/integration/operatorhub/cmd_service_test.go
@@ -88,6 +88,7 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
                        if err := ioutil.WriteFile(fileName, []byte(stdOut), 0644); err != nil {
                                fmt.Printf("Could not write yaml spec to file %s because of the error %v", fileName, err.Error())
                        }
+                       helper.CmdShouldPass("sed", "-i", "s/example/example2/", fileName)
 
                        // now create operator backed service
                        helper.CmdShouldPass("odo", "service", "create", "--from-file", fileName)
@@ -95,7 +96,7 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
                        // now verify if the pods for the operator have started
                        pods := helper.CmdShouldPass("oc", "get", "pods", "-n", CI_OPERATOR_HUB_PROJECT)
                        // Look for pod with example name because that's the name etcd will give to the pods.
-                       etcdPod := regexp.MustCompile(`example-.[a-z0-9]*`).FindString(pods)
+                       etcdPod := regexp.MustCompile(`example2-.[a-z0-9]*`).FindString(pods)
 
                        ocArgs := []string{"get", "pods", etcdPod, "-o", "template=\"{{.status.phase}}\"", "-n", CI_OPERATOR_HUB_PROJECT}
                        helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
```

Reply:
This will fail on windows I think

Reply:
@mik-dass 

> Can't we use different values for those tests?

To answer this more accurately, this data is obtained on the fly. We're neither using static data nor reliable data. A newer version of the operator (etcdoperator in this case) could lead to a different yaml that could break entire test suite (exaggerating a bit but it's possible.) So the best way to mitigate this would be to write our own operator with testing in mind.

Reply:
> This will fail on windows I think. Maybe use a golang library?

Sure, I'll open a new issue and assign it to you. :stuck_out_tongue_closed_eyes:

Reply:
> I tried `strings.Replace` but that's messing up the yaml formatting. So the next option could be to do yaml unmarshalling. It's a pain, tbh.

This mess was due to my mistake. Made the changes.

Reply:
> Sure, I'll open a new issue and assign it to you. stuck_out_tongue_closed_eyes

No problem I like windows :D